File: Iniciante/URI 1013.hs
main :: IO ()
main = do
    line <- getLine
    let (valor1:valor2:valor3:_) = map (read :: String -> Int) $ words line

    if valor1 > valor2 && valor1 > valor3
        then putStrLn $ show valor1 ++ " eh o maior"
    else if valor2 > valor3
        then putStrLn $ show valor2 ++ " eh o maior"
    else
        putStrLn $ show valor3 ++ " eh o maior"

File: Iniciante/URI 1013.lua
line = {}
n = 0
for i in string.gmatch(io.read("*l"), "%S+") do
   n = n + 1
   line[n] = i
end
valor1 = tonumber(line[1])
valor2 = tonumber(line[2])
valor3 = tonumber(line[3])

if (valor1 > valor2 and valor1 > valor3) then
	print(valor1 .. ' eh o maior')
elseif (valor2 > valor3) then
	print(valor2 .. ' eh o maior')
else 
	print(valor3 .. ' eh o maior')
end